.github/workflows/docker.yml

```
@@ -1,116 +1,113 @@
 
 jobs:
-  build_docker:    
-    name: Build Docker
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+    build_docker:
+        name: Build Docker
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)